src/arithmetic/numeric_funcs/numeric_funcs.h

```diff
@@ -10,3 +10,6 @@
 
 void Register_NumericFuncs();
 
+SIValue AR_TOFLOAT(SIValue *argv, int argc, void *private_data);
```

Comment:
Formatting:
```c
// tries to convert input to float
SIValue AR_TOFLOAT
(
    SIValue *argv,         // arguments 
    int argc,                    // number of arguments
    void *private_data // private context
);
```

Reply:
Format fixed in [831f6c0](https://github.com/RedisGraph/RedisGraph/pull/2605/commits/831f6c039219369ec38266dd2b35b027e8d71594)